rs/nns/governance/src/neurons_fund.rs

```diff
@@ -1075,26 +1075,32 @@ where
         // `neurons_fund_reserves.total_amount_icp_e8s()`, the sum of (pre-rounded) `u64` values.
         let (neurons_fund_reserves, allocated_neurons_fund_participation_icp_e8s) =
             if total_maturity_equivalent_icp_e8s == 0 {
-                println!(
-                    "{}WARNING: Neurons' Fund has zero total maturity.",
-                    governance::LOG_PREFIX
-                );
+                #[cfg(not(test))]
+                {
+                    println!(
+                        "{}WARNING: Neurons' Fund has zero total maturity.",
+                        governance::LOG_PREFIX
+                    );
+                }
```

Comment:
There is no `use dfn_core::println` at the top? If so, this actually means the logs are not printed in production.

I believe `use dfn_core::println` achieves roughly the same thing as what's done here. In unit tests (not compiled as wasm), those will not be printed, while in state machine tests, pocket ic tests or production, those will be printed. It's possible that we want to disable the logging in state machine tests and pocket ic tests, but I believe the `#[cfg(not(test))]` blocks are also enabled in those tests.

Reply:
Yeah this is a good point.  

Maybe the logs should just be removed instead?

Reply:
(It's easy to forget to set up an NF in testing, and then spend time understanding why it didn't participate in an SNS)

Reply:
```suggestion
```